Library/Homebrew/test/test_formula_install.rb

```diff
@@ -49,13 +49,4 @@ def test_a_basic_install
       assert_equal 3, (HOMEBREW_PREFIX+'bin').children.length
     end
   end
-
-  def test_script_install
-    f = Class.new(ScriptFileFormula) do
-      url "file://#{File.expand_path(__FILE__)}"
-      version "1"
-    end.new("test_script_formula", Pathname.new(__FILE__).expand_path, :stable)
-
-    temporary_install(f) { assert_equal 1, f.bin.children.length }
-  end
```

Comment:
Since we are deprecating but not yet removing this functionality, we should preserve the test coverage.